com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariable.cs

```diff
@@ -66,6 +66,14 @@ public NetworkVariable(T value)
         [SerializeField]
         private T m_InternalValue;
 
+        /// <summary>
+        /// The temporary accessor to enable struct element access until [MTT-1020] complete
+        /// </summary>
+        public ref T ValueRef
+        {
+            get => ref m_InternalValue;
+        }
```

Comment:
this part is introduced by PR #1051 originally, it's here too.
I'm not sure about the ultimate API, @mattwalsh-unity might chime in here as he wants to drive this change/shift around the codebase.

Reply:
This is commented temporary, granted, but I'm not familiar with the linked issue. Can you elaborate on why marking the netvar dirty is not needed here ?

Reply:
I'm not entirely sure what is the new design direction but I can tell you why NetworkTransform's State still works: we manually mark NetVar to be dirty later in code. So, we don't rely on a get/set prop marking NetVar dirty, we mark the NetVar dirty somewhere else.